src/js/smooch.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { render, unmountComponentAtNode } from 'react-dom';
 import pick from 'lodash.pick';
+import { batchActions } from 'redux-batched-actions';
 
 import { store } from './stores/app-store';
```

Comment:
> only if a Smooch method needs it or when it needs to render

Curious in which case you can have the widget load without the store? Just calling `.init()` will trigger you to load the store, right?